### ISSUE TITLE
Build Fails on Arch Linux "Requires Bazel 6.1.0"

### ISSUE THREAD

**Issue by 7t2** [rocm-arch/tensorflow-rocm#69]
When installing tensorflow-rocm on arch linux, after being asked if I would like to interactively configure ./WORKSPACE for Android builds, I get the following error every time:

ERROR: The project you're trying to build requires Bazel 6.1.0 (specified in /home/usrname/.cache/yay/tensorflow-rocm/src/tensorflow-2.15.0-rocm/.bazelversion), but it wasn't found in /usr/bin.

Even if I download the required bazel release and put it in the location it requests, the build still fails with the following error:

ERROR: no such package '@local_config_rocm//rocm': Repository command failed

I am totally stumped, does anyone know what needs to be done?

**Comment by uberkael** [comment]
@7t2 I had the same error and fixed it by installing `bazelisk`

But now the error is : 
```
ERROR: An error occurred during the fetch of repository 'local_config_rocm':
   Traceback (most recent call last):```

**Comment by 7t2** [comment]
Thanks for the heads up with bazelisk, although I get stuck at that same error too now. I suspect something is very wrong with the rocm and tensorflow stack on arch linux at the moment.

**Comment by uberkael** [comment]
Yes there is this other issue for it #68

**Comment by acxz** [comment]
This particularly issue (requires bazel 6.1.0) is resolved with [fa64355](https://github.com/rocm-arch/tensorflow-rocm/commit/fa64355329e611aa83109f97e68128aafccb50b3)
